src/core/Slider/Thumb.tsx

```diff
@@ -90,9 +90,11 @@ export const Thumb = (props: ThumbProps) => {
       }
       switch (event.key) {
         case 'ArrowLeft':
+        case 'ArrowDown':
           onThumbValueChanged(index, Math.max(value - step, minVal));
           return;
         case 'ArrowRight':
+        case 'ArrowUp':
```

Comment:
Not sure about this change. Up is not the same as Right, and Down is not the same as Left.

Reply:
These new arrow cases were added to mimic what was supported in react-compound-slider. I actually ran into a unit test that used the down arrow to move thumb to left (ie lower the thumb value).

Reply:
Okay, I checked the native slider and W3 variant and they both use Up/Down too, so this is fine.

I would add return early if `altKey` is pressed though. See example in https://github.com/iTwin/iTwinUI-react/blob/main/src/core/Menu/MenuItem.tsx#L107

Reply:
done